CONTRIBUTING.md

```diff
@@ -96,6 +96,22 @@ The TL;DR; of the above very precise workflow version is:
 6. Keep polishing it until received enough LGTM
 7. Profit!
 
+### Backporting
+Backport pull requests such as these are marked using the phrase `for validation` in the title to make the purpose clear in the pull request list. 
+They can be merged once validation passes without additional review (if there are no conflicts). 
+Using, for example: current.version 2.5.22, previous.version 2.5, milestone.version 2.6.0-M1    
+1. Label this PR with `to-be-backported`
+1. Mark this PR with Milestone `${milestone.version}`
+1. Mark the issue with Milestone `${current.version}`
+1. `cherry-pick to release-${previous.version}`
+1. `git checkout release-${previous.version}`
+1. `git pull`
+1. `git cherry-pick <commit>`
```

Comment:
Maybe someone may wrongly interpret this as the cherry-pick is commited to the release-2.5 branch here. Maybe add a ”create wip branch” before this step?

Reply:
Good point.